admin/worker/run_autoscaler.go

```diff
@@ -48,13 +56,21 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 			continue
 		}
 
-		if !shouldScale(targetProject.ProdSlots, rec.RecommendedSlots) {
-			w.logger.Debug("skipping autoscaler: target slots are within threshold of original slots",
+		if shouldScale, reason := shouldScale(targetProject.ProdSlots, rec.RecommendedSlots); !shouldScale {
+			logMessage := "skipping autoscaler: " + reason
+
+			logFields := []zap.Field{
 				zap.Int("project_slots", targetProject.ProdSlots),
 				zap.Int("recommend_slots", rec.RecommendedSlots),
 				zap.Float64("scale_threshold_percentage", scaleThreshold),
 				zap.String("project_name", targetProject.Name),
-			)
+			}
+
+			if reason == scaledown {
+				w.logger.Info(logMessage, logFields...)
```

Comment:
@nishantmonu51 for "not scaling down", the code here is designed to leave a message so that we can later create a datadog monitor to capture the log pattern with this.